src/material-experimental/mdc-theming/_all-theme.scss

```diff
@@ -6,7 +6,6 @@
 @import '../mdc-menu/menu-theme';
 @import '../mdc-radio/radio-theme';
 @import '../mdc-slide-toggle/slide-toggle-theme';
-@import '../mdc-slider/slider-theme';
```

Comment:
Wouldn't this prevent us from catching any bugs in the dev app until MDC are done updating their slider? It might be better to leave it as it is and then swap it out when MDC have released the rewrite.